templateflow/client.py

```diff
@@ -0,0 +1,479 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2024 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+"""TemplateFlow's Python Client."""
+
+from __future__ import annotations
+
+import os
+import sys
+from json import loads
+from pathlib import Path
+
+from .conf.cache import CacheConfig, TemplateFlowCache
```

Comment:
Last time we discussed (if I remember properly), we decided to use absolute imports. Is that correct? Has there been some update to best practices on the direction of relative imports?

Otherwise:


```suggestion
from templateflow.conf.cache import CacheConfig, TemplateFlowCache
```